### ISSUE TITLE
Make possible to disable the annoying "This dashboard is currently force refreshing" message

### ISSUE THREAD

**Issue by ktecho** [apache/superset#13242]
**Is your feature request related to a problem? Please describe.**

With Superset 1.0, it seems a new toast has been added for when you have auto-refresh in a dashboard:

"This dashboard is currently force refreshing; the next force refresh will be in 10 seconds."

This is super-annoying because it distracts your attention from the changing dashboard. I think the "infinite / superset" animated symbol in each graph is enough. Besides, if you go to another tab in your browser, sometimes you can see more than one toast stacked.

**Describe the solution you'd like**

If you think it's a good idea to have the toast displaying each 10 seconds (maybe once at first would be fine...), please make it configurable so I can disable it from the UI or directly in the json config.

**Describe alternatives you've considered**

One alternative would be displaying the message just once when you open a dashboard.

**Comment by junlincc** [comment]
> if you go to another tab in your browser, sometimes you can see more than one toast stacked.

Thanks for reporting. I sometimes find the toast massage taking away the focus and annoying too(as the PM😅). agree that one loading-infinity icon should be sufficient to indicate refresh activity. 
@steejay wdyt?

https://user-images.githubusercontent.com/67837651/108571307-c178ad80-72c4-11eb-8714-c20fd7963500.mov

**Comment by Steejay** [comment]
hmm interesting. i think displaying the toast every 10 seconds is excessive. its like an alarm clock you cant turn off in the morning. how do you turn on auto refresh? is it through the Superset UI? If so, we can add some information in the location of that control via the use of help text. Therefore when the user turns on auto refresh theyll understand the result of this action in context. 

Another potential alternative would be to include a "last updated" indicator somewhere on the dashboard.

**Comment by ktecho** [comment]
Yes, I like @Steejay suggestion about including a "last updated" indicator. But even without that, you'll see that the dashboard is auto-refreshing because the widgets refreshes with the animated infinite symbol.

**Comment by Steejay** [comment]
@ktecho agreed. it would be good to consider the users who might experience the dashboard too. for the dashboard creator the auto refresh cadence might be well understood but for someone who only observes the dashboard, they might not have context on how often the dashboard is refreshing unless they're counting the intervals. are we willing to accept this potential ambiguity? if so the loading symbol might be enough as is. if not a "last updated" indicator could add context while still staying relatively subtle.

**Comment by ktecho** [comment]
Yeah, totally agree.

**Comment by junlincc** [comment]
we had a discussion about adding a status bar to show some basic info and dashboard interaction setting in the dashboard view, which could also include 'last refresh'. 

@ktecho can we agree that "last updated" indicator is still necessary for dashboard consumer, but not necessary in a form of the toast msg?

**Comment by ktecho** [comment]
@junlincc Yeah, I think that would make it.

**Comment by Maximilian-Pichler** [comment]
+1

**Comment by xallaume** [comment]
+1

**Comment by ga2arch** [comment]
+1

**Comment by johhnydinh** [comment]
+1
